cmake/LLVMIRUtil.cmake

```diff
@@ -175,16 +175,17 @@ function(llvmir_attach_bc_target)
       continue()
     endif()
 
+    get_target_property(SOURCE_DIR ${DEPENDS_TRGT} SOURCE_DIR)
+    get_filename_component(INFILE ${IN_FILE} ABSOLUTE BASE_DIR ${SOURCE_DIR})
```

Comment:
Can we unify the use of `INFILE` with `IN_FILE`?
I find the current naming confusing.